### PR TITLE
Adds reconnect button

### PR DESCRIPTION
**Pull request by Robertha89**
image
:cl: PPI
rscadd: Adds a reconnect button to the File menu in the client.
/:cl:

### CODE REVIEW

**Comment by Gummin** [comment]
There already is a reconnect button. (but it's buggy as hell.)

**Comment by Robertha89** [comment]
the old reconnect button was removed when control freak was added.

**Comment by Aurorablade** [comment]
right click title bar...get options....or should..normally...

**Comment by Robertha89** [comment]
yeah, like i said, control freak removed this for us plebs, so it's only for certain admins, the adminiest of admins.

**Comment by NTSAM** [comment]
Thank you.
:+1:

**Comment by Fox-McCloud** [comment]
port of: https://github.com/tgstation/-tg-station/pull/12412

File: interface/skin.dmf [hunk 1]
```diff
@@ -1375,6 +1375,15 @@ menu "menu"
 		is-disabled = false
 		saved-params = "is-checked"
 	elem 
+		name = "Reconnect"
```
Comment (TheDZD) [inline interface/skin.dmf]: Given the fact that every other element has its name set as `&<thing>`, this should probably be `&Reconnect`.

**Comment by Robertha89** [comment]
@Fox-McCloud Not a port, i made it in my skin.dm and re-used the code it made here.

**Pull request merged**